rakefile.netstd.rb

```diff
@@ -48,15 +49,13 @@
 ]
 
 repo = 'FakeItEasy/FakeItEasy'
-teamcity_server_url = 'http://teamcity.codebetter.com/admin/editBuildParams.html?id=buildType:bt929'
 appveyor_server_url = 'https://ci.appveyor.com/project/FakeItEasy/fakeiteasy/settings'

```

Comment:
I went ahead and did the AppVeyor project delete and rename and I change the URL used in README.md to match the one use here (the pretty one).

I also squashed all the commits. Hopefully this PR is ready to go now.

Oh, and I rebased too.